### PR TITLE
AKASH ANI.md

### PR DESCRIPTION
**Pull request by akash-1724**

### CODE REVIEW

**Review by rajatsandeepsen — CHANGES_REQUESTED** [review]
Cool ✨ 

But can you use less # in text...

**Comment by alvin-dennis** [comment]
@akash-1724 Pls work on ur '#' usage and redo it

**Comment by akash-1724** [comment]
Changed

File: participants/AKASH ANI.md [hunk 1]
```diff
@@ -0,0 +1,58 @@
+# Akash Ani
+
+# About Me
+
+
+# Hello! I’m Akash, a passionate Computer Science Engineering student at St. Joseph’s Engineering and Technology, Palai. I thrive on learning new skills and have a special affinity for mathematics and coding. My curiosity extends to the fascinating world of data science. Proficient in both Python and C programming languages, I’m always eager to explore and create. 🚀
```
Comment (rajatsandeepsen) [inline participants/AKASH ANI.md:6]: Too much #

**Review by rajatsandeepsen — APPROVED** [review]
But cool

**Pull request closed without merging**